docs/src/pages/WebSwitchPage/WebSwitchPage.js

```diff
@@ -43,6 +43,22 @@ const components = [
       </div>,
     ],
   },
+  {
+    id: 'event',
+    title: 'Event',
+    blurb: [
+      <Paragraph>
+        Reserved for event-based UI that has neither a positive or negative
+        connotation, for example fare flexibility brought on by a global event,
+        or a weather event.
```

Comment:
Can you make it use the same text as the banner alert? It's my bad for not updating the card, not your fault:

> Reserved for event-based messaging that has neither a positive or negative connotation. Avoid using this in other contexts.

Reply:
Done!